benches/src/print.rs

```diff
@@ -0,0 +1,205 @@
+use std::collections::HashMap;
+use std::io::Write;
+
+use termcolor::{Color, ColorSpec, StandardStream, WriteColor};
+
+use crate::SubBenchResult;
+
+#[derive(Default, Clone)]
+struct Delta {
+    throughput_change_str: String,
+    throughput_change: f64,
+    time_change_str: String,
+    time_change: f64,
+}
+
+impl Delta {
+    fn new(previous: &SubBenchResult, current: &SubBenchResult) -> Self {
+        let prev_throughput = previous.throughput_per_second();
+        let curr_throughput = current.throughput_per_second();
+        let delta_throughput = if prev_throughput != 0.0 {
+            (curr_throughput - prev_throughput) / prev_throughput * 100.0
+        } else {
+            0.0
+        };
+        let throughput_change = format!(" ({delta_throughput:+.2}%)");
+
+        let prev_time = previous.avg_duration_per_iteration;
+        let curr_time = current.avg_duration_per_iteration;
+        let delta_time = if prev_time.as_nanos() != 0 {
+            (curr_time.as_secs_f64() - prev_time.as_secs_f64()) / prev_time.as_secs_f64() * 100.0
+        } else {
+            0.0
+        };
+
+        let time_change = format!("{delta_time:+.2}%; ");
+
+        Delta {
+            throughput_change_str: throughput_change,
+            throughput_change: delta_throughput,
+            time_change_str: time_change,
+            time_change: delta_time,
+        }
+    }
+}
+
+/// Get a color spec for the given change value.
+///
+/// Positive changes are red (regression), negative changes are green (improvement).
+/// This represents changes for time durations. For throughput changes, the sign should be inverted.
+fn get_change_color(change: f64) -> ColorSpec {
```

Comment:
those are changes in percent? why not documented or named like that

Reply:
Yeah - wasn't that way before